app/controllers/master_files_controller.rb

```diff
@@ -325,6 +325,11 @@ def hls_manifest
     end
   end
 
+  def caption_manifest
+    return head :unauthorized if cannot?(:read, @master_file)
+    @caption = @master_file.captions
```

Comment:
This line is unnessary for final implementation. The `caption_manifest` erb file does not acually make use of this but rather uses the `captions_master_file_path`. Shouldn't interfere with testing though.